extensions/mssql/src/objectManagement/ui/databaseDialog.ts

```diff
@@ -49,10 +65,23 @@ export class DatabaseDialog extends ObjectManagementDialogBase<Database, Databas
 			}
 			this.formContainer.addItems(components);
 		} else {
+			// Options of a boolean dropdown
+			this.booleanOptionsArray = Object.keys(BooleanDropdownOptions);
 			// Initilaize general Tab sections
 			this.initializeBackupSection();
 			this.initializeDatabaseSection();
 
+			//Initilaize options Tab sections
+			this.initializeOptionsGeneralSection();
+			this.initializeAutomaticSection();
+			if (this.objectInfo.isLedgerDatabase !== null) {
+				this.initializeLedgerSection();
+			}
+			if (this.objectInfo.pageVerify !== null && this.objectInfo.targetRecoveryTimeInSec !== null) {
```

Comment:
These ones are a little tricker since 0 or empty strings might be valid options. So at the very least for these you should be doing a "null or undefined" check which we have a helper function for here : https://github.com/microsoft/azuredatastudio/blob/5b0f488cd5ddbf5f954d22be28d9604da6140080/extensions/mssql/src/types.ts#L17